traffic_ops_ort/t3c/config/config.go

```diff
@@ -218,6 +221,9 @@ func GetCfg() (Cfg, error) {
 	dnsLocalBindPtr := getopt.BoolLong("dns-local-bind", 'b', "[true | false] whether to use the server's Service Addresses to set the ATS DNS local bind address")
 	helpPtr := getopt.BoolLong("help", 'h', "Print usage information and exit")
 	useGitStr := getopt.StringLong("git", 'g', "auto", "Create and use a git repo in the config directory. Options are yes, no, and auto. If yes, create and use. If auto, use if it exist. Default is auto.")
+	syncdsUpdatesIPAllowPtr := getopt.BoolLong("syncds-updates-ipallow", 'S', "Whether syncds mode will update ipallow. This exists because ATS had a bug where reloading after changing ipallow would block everything. Default is false.")
+	omitViaStringReleasePtr := getopt.BoolLong("omit-via-string-release", 'v', "Whether to set the records.config via header to the ATS release from the RPM. Default true.")
+	disableParentConfigCommentsPtr := getopt.BoolLong("disable-parent-config-comments", 'c', "Whether to disable verbose parent.config comments. Default false.")
 	defaultEnableH2 := getopt.BoolLong("default-client-enable-h2", '2', "Whether to enable HTTP/2 on Delivery Services by default, if they have no explicit Parameter. This is irrelevant if ATS records.config is not serving H2. If omitted, H2 is disabled.")
 	defaultClientTLSVersions := getopt.StringLong("default-client-tls-versions", 'v', "", "Comma-delimited list of default TLS versions for Delivery Services with no Parameter, e.g. --default-tls-versions='1.1,1.2,1.3'. If omitted, all versions are enabled.")
```

Comment:
The ORT integration tests GHA fails because the `-v` flag is used twice:

```go
2021-05-10T17:05:35.7619221Z     t3c-dns-local-bind_test.go:38: ERROR: t3c badass failed: exit status 1: stdout:  stderr: /tmp/go/src/github.com/apache/trafficcontrol/traffic_ops_ort/t3c/config/config.go:228: -v already declared at /tmp/go/src/github.com/apache/trafficcontrol/traffic_ops_ort/t3c/config/config.go:225
2021-05-10T17:05:35.7622426Z         
2021-05-10T17:05:38.5006065Z     types.go:58: ---- DeleteTestTypes ----
2021-05-10T17:05:38.5006834Z User: traffic_ops, Password: twelve, Hostname: db, DB: traffic_ops
2021-05-10T17:15:32.7954658Z panic: test timed out after 10m0s
2021-05-10T17:15:32.7962891Z 
2021-05-10T17:15:32.7963360Z goroutine 85 [running]:
2021-05-10T17:15:32.7965135Z testing.(*M).startAlarm.func1()
2021-05-10T17:15:32.7965696Z 	/usr/local/go/src/testing/testing.go:1618 +0xe5
2021-05-10T17:15:32.7966348Z created by time.goFunc
2021-05-10T17:15:32.7966986Z 	/usr/local/go/src/time/sleep.go:167 +0x45
2021-05-10T17:15:32.7967267Z 
2021-05-10T17:15:32.7967667Z goroutine 1 [chan receive, 10 minutes]:
2021-05-10T17:15:32.7968206Z testing.(*T).Run(0xc000232480, 0xa4e88c, 0x13, 0xa7d058, 0x496306)
2021-05-10T17:15:32.7968764Z 	/usr/local/go/src/testing/testing.go:1169 +0x2da
2021-05-10T17:15:32.7969353Z testing.runTests.func1(0xc000232300)
2021-05-10T17:15:32.7970069Z 	/usr/local/go/src/testing/testing.go:1439 +0x78
2021-05-10T17:15:32.7970594Z testing.tRunner(0xc000232300, 0xc000243878)
2021-05-10T17:15:32.7971094Z 	/usr/local/go/src/testing/testing.go:1123 +0xef
2021-05-10T17:15:32.7971765Z testing.runTests(0xc00020c620, 0xe25580, 0x5, 0x5, 0xc01e786d2f38b694, 0x8bbdf5f0f3, 0xe35220, 0x0)
2021-05-10T17:15:32.7972419Z 	/usr/local/go/src/testing/testing.go:1437 +0x2fe
2021-05-10T17:15:32.7972893Z testing.(*M).Run(0xc000074100, 0x0)
2021-05-10T17:15:32.7973380Z 	/usr/local/go/src/testing/testing.go:1345 +0x1eb
2021-05-10T17:15:32.7974967Z github.com/apache/trafficcontrol/traffic_ops_ort/testing/ort-tests.TestMain(0xc000074100)
2021-05-10T17:15:32.7976246Z 	/root/go/src/github.com/apache/trafficcontrol/traffic_ops_ort/testing/ort-tests/traffic_ops_ort_test.go:115 +0x916
2021-05-10T17:15:32.7976922Z main.main()
2021-05-10T17:15:32.7977451Z 	_testmain.go:53 +0x165
2021-05-10T17:15:32.7977875Z 
2021-05-10T17:15:32.7978235Z goroutine 5 [select, 10 minutes]:
2021-05-10T17:15:32.7978855Z database/sql.(*DB).connectionOpener(0xc0003368f0, 0xb07b80, 0xc000067bc0)
2021-05-10T17:15:32.7979468Z 	/usr/local/go/src/database/sql/sql.go:1126 +0xf5
2021-05-10T17:15:32.7981230Z created by database/sql.OpenDB
2021-05-10T17:15:32.7981799Z 	/usr/local/go/src/database/sql/sql.go:740 +0x12a
2021-05-10T17:15:32.7999839Z exit status 2
2021-05-10T17:15:32.8001262Z FAIL	github.com/apache/trafficcontrol/traffic_ops_ort/testing/ort-tests	600.198s
2021-05-10T17:15:32.8398667Z ERROR: ORT tests failure
2021-05-10T17:15:33.0119381Z 3
2021-05-10T17:15:33.0859032Z Child process " docker-compose -f /home/runner/work/trafficcontrol/trafficcontrol/traffic_ops_ort/testing/docker/docker-compose.yml run ort_test " exited with status code 3 !
```

Reply:
Ah, another PR added that after I made this one. Fixed.